server/routerlicious/packages/kafka-orderer/src/kafkaOrderer.ts

```diff
@@ -40,26 +38,21 @@ export class KafkaOrdererConnection implements core.IOrdererConnection {
             serviceConfiguration);
     }
 
-    public get parentBranch(): string {
-        return this._parentBranch;
+    // TODO remove after client packages update
```

Comment:
I'd rather put // back-compat comment, as at least we scan them periodically :) I do not think anybody scans TODO comments. Reference to issue would be even better :)

Reply:
Updated in latest, opened and referenced #4346